src/source/Signaling/LwsApiCalls.c

```diff
@@ -689,6 +693,11 @@ STATUS describeChannelLws(PSignalingClient pSignalingClient, UINT64 time)
     pResponseStr = pLwsCallInfo->callInfo.responseData;
     resultLen = pLwsCallInfo->callInfo.responseDataLen;
 
+    if (!isLwsCallResultFailureRetryable(&pLwsCallInfo->callInfo)) {
```

Comment:
Minor: Perhaps cleaner to use:

CHK_ERR(isCallResultFailureRetryable(&pLwsCallInfo->callInfo, STATUS_SIGNALING_DESCRIBE_CALL_FAILED, "Describe API call failed with %*.s and will not be retried.", resultLen, pResounseStr);

Reply:
Yeah I thought of that but I chose to not do that to avoid a duplicate log, I added a general error log in case the retStatus was not success, but I think including something specific to the call not being retried is fine, I made this change we can live with 2 log messages.